corehq/apps/ota/views.py

```diff
@@ -37,6 +38,7 @@
 from .utils import demo_user_restore_response, get_restore_user, is_permitted_to_restore, handle_401_response
 
 
+@location_safe
```

Comment:
I *think* it should work as long as the other decorators use `@wraps`.  It sets an attr on the function itself, and some decorators return a different, anonymous function.